### ISSUE TITLE
Status of touch devices — iPad does not work

### ISSUE THREAD

**Issue by lab-at-nohl** [tine20/tine20#6931]
Which touch commands can the web ui handle? I‘m stuck on an iPad right now and I can‘t move mails to folders or similiar. A recent Android shows a context menu on a „long“ touch.

**Comment by lab-at-nohl** [comment]
Could someone with a „full“ Mac-equipment go for this bug?

I don‘t have a Mac Desktop which is - as it seems - required for debugging websites on iPad. There is no internal debug console or similar on iOS?!

**Comment by msphn** [comment]
@lab-at-nohl yes, there is no. You need to use safari with a Mac. Then you can select your iPad as device in the safari dev tools.

**Comment by lab-at-nohl** [comment]
> You need to use safari with a Mac. 

...which I don‘t have. All I could do is googling, and it seems other projects have this problem on iOS > 11.x, too. @msphn could you mark this as a bug?

**Comment by msphn** [comment]
I guess the problem is the iPhone XR, it won't have 3d touch and they implementing replacements for that. Wouldn't be surprised if it's an actual iOS bug. Marked it as a bug for now!

**Comment by pschuele** [comment]
@corneliusweiss could you please have a look at this?

**Comment by corneliusweiss** [comment]
the magic trick is to use two fingers touch for context menus. For a mac user quite normal to user two fingers on the touchpad to get context menu.

force-touch would be an alternativ option but we havn't implemented it yet - and it might not be worth to do so @see https://www.heise.de/mac-and-i/meldung/Apple-beerdigt-angeblich-3D-Touch-4146751.html

attachments: we don't allow attachment download atm. as touch devices tend to send downloads to their configured clouds. Feel free to open a FR to put this behind a config option/preference

**Comment by lab-at-nohl** [comment]
Ah, good to know. But in reality, my fingers are to thick. It always selects one menu item when using it. It is not a real problem but could you add some kind of delay, quarte second or so? 

The thing with the attachments irritates me - how do you open e.g. a pdf file (could be handled by internal viewer)?

But thank you for the help!
